bundles/thunk_bundle.js

```diff
@@ -4300,6 +4300,9 @@ module.exports = (() => {
   init_buffer_shim();
   var import_clone = __toESM(require_clone());
 
+  // helpers/object_utils.ts
+  init_buffer_shim();
```

Comment:
Any idea why this changed?

Reply:
I am not sure. I tried make bs and rebase and nothing was changed. I can play with the revision history a bit before merging this.

Reply:
oh turns out whenever we import something `testing/injections/buffer_shim.js` will be injected and that leads to `init_buffer_shim`. 

it's still a bit weird why `dist/bundles/thunk_bundle.js` is adding that line. could be an esbuild feature or that we are not using esbuild inject properly. but I can't quickly find a way to avoid this line.